### PR TITLE
Callcenter UCR queries

### PR DESCRIPTION
**Pull request by snopoke**
builds on top of #11315 
@czue 
buddy @dannyroberts

### CODE REVIEW

File: corehq/apps/callcenter/queries.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+        query = select(
+            self.columns(include_type_in_result, distinct_docs=distinct_docs)
+        ).where(and_(
+            *all_where
+        )).group_by(
+            *self.group_by(include_type_in_result)
+        )
+        return query
+
+class CaseQueryOpenedClosed(CaseQuery):
```
Comment (dimagimon) [inline corehq/apps/callcenter/queries.py]: E302 expected 2 blank lines, found 1

**Comment by dimagimon** [comment]
There are 6 errors:

* corehq/apps/callcenter/app_parser.py, line 4 - F811 redefinition of unused 'CallCenterIndicatorConfig' from line 4
* corehq/apps/callcenter/tests/test_use_fixtures_configuration.py, line 98 - E231 missing whitespace after ','
* corehq/apps/callcenter/tests/test_use_fixtures_configuration.py, line 33 - W293 blank line contains whitespace
* corehq/apps/callcenter/models.py, line 30 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/callcenter/tests/test_use_fixtures_configuration.py, line 48 - W293 blank line contains whitespace
* corehq/apps/callcenter/tests/test_models.py, line 96 - E501 line too long (120 > 115 characters)

File: corehq/apps/callcenter/queries.py (hunk 1 above)
Comment (dimagimon) [inline corehq/apps/callcenter/queries.py]: E302 expected 2 blank lines, found 1

**Comment by czue** [comment]
did you want to reopen this against master? let me know if ready for review

**Pull request closed without merging**